x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks_table.tsx

```diff
@@ -67,6 +78,39 @@ const IntegrationButtonLink = ({
   );
 };
 
+const ErrorMessageComponent = (error: ErrorProps) => (
```

Comment:
Neat: there's no need to declare types when they are one-liners and used only in one function.

```js
const ErrorMessageComponent = (error: unknown) => (
```